src/pages/games/story-game/index.jsx

```
@@ -1,36 +1,310 @@
-import React from 'react';
-// import GameCompleteStory from '../../../components/games/story-game/game-complete-story';
-import GameCompleteStory from '@/components/Games/story-game/game-complete-story'
-
+import React, { useState, useEffect } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)